README.md

```diff
@@ -574,6 +594,25 @@ targets = {
 Requires:
 * Python [dbus](http://www.freedesktop.org/wiki/Software/DBusBindings/#Python) bindings
 
+### `discord`
+
+The `discord` service provides notifications through discords webhooks.
+
+![Discord Webhook](assets/discord.png)
```

Comment:
Are you aware that your screenshot displays `mmqt` several times? This is not a problem for me, just pointing out. (I assume you mean `mqtt` :-) )

Reply:
I had two channels for mqtt, didn't notice the type thanks.